packages/select/src/ResourceSelect.js

```diff
@@ -173,12 +174,20 @@ const ResourceSelect = ({
     } else {
       fetch = () =>
         resource.postGet(
-          qs.stringify(params, {
-            encode: false,
-            arrayFormat: 'repeat',
-            indices: false,
-            allowDots: true,
-          }),
+          // query-string does not support nesting values, convert additionalPostGetArgs
+          // to JSON string to be parsed by API
+          qs.stringify(
+            {
+              ...params,
+              additionalPostGetArgs: JSON.stringify(additionalPostGetArgs),
```

Comment:
`additionalPostGetArgs` should be the third argument to `postGet`. not part of what gets `qs.stringified`.